webapi/scripts/graphdb-se-knora-test-repository-config.ttl.tmpl

```diff
@@ -17,6 +17,6 @@
        owlim:repository-type "file-repository" ;
        owlim:query-timeout "30" ;
        owlim:throw-QueryEvaluationException-on-timeout "true";
-       owlim:query-limit-results "50000";
+       # owlim:query-limit-results "50000";
```

Comment:
Do you think there is no limit we could define to protect the triplestore? Now with standoff queried in iterations, the previous limit should not be  a problem. I agree that such a limit will always be somehow arbitrary.